cli/config.go

```diff
@@ -15,8 +15,9 @@ type Config struct {
 }
 
 type ConfigContext struct {
-	Name  string
-	Token string
+	Name    string
+	Token   string
+	Current string
```

Comment:
That field doesn’t belong into the model as it’s specific to presentation.

Reply:
I have changed it to a specific model for the presentation.

Reply:
👏